docs/how/updating-datahub.md

```diff
@@ -5,12 +5,12 @@ This file documents any backwards-incompatible changes in DataHub and assists pe
 ## Next
 
 ### Breaking Changes
-- Remove `database` option from `snowflake` source which was deprecated since a few versions
-- Rename confusing config `report_upstream_lineage` to `upstream_lineage_in_report` in `snowflake` connector which was added in `0.8.32`
+- #4644 Remove `database` option from `snowflake` source which was deprecated since a few versions
```

Comment:
Can we specify the version number instead of saying "since few versions"?

Reply:
done